test/jobs/zendesk_job_test.rb

```diff
@@ -0,0 +1,61 @@
+require_relative '../test_helper'
+
+SingleCov.covered!
+
+describe 'Zendesk Job' do
+  hostname = 'https://hostname.com'
+  username = 'username@email.com'
+  password = '123456'
+  describe 'boostraps zendesk job' do
+    it 'bootstraps by getting environment variables' do
+      ENV['ZENDESK_HOSTNAME'] = hostname
+      ENV['ZENDESK_USERNAME'] = username
+      ENV['ZENDESK_PASSWORD'] = password
+      ZendeskJob.bootstrap()
+      assert ZendeskJob.hostname == hostname
+      assert ZendeskJob.username == username
+      assert ZendeskJob.password == password
```

Comment:
use `assert_equal expected, actual` here, otherwise the error message will just say "Assertion failed, no message given." which isn't very helpful.